assets/css/custom.css

```diff
@@ -0,0 +1,145 @@
+.img-c {
+    width: 100%;
+    height: 18.875rem;
+    object-fit: cover;
+}
+
+.img-sm {
+    width: 100%;
+    height: 10.5rem;
+    object-fit: cover;
+}
+
+.img-xs {
+    width: 100%;
+    height: 7.5rem;
+    object-fit: cover;
+}
+
+.window-100{
+    width: 70%;
+}
+
+.window-80{
+    width: 80%;
+}
+
+.window-70{
+    width: 70%;
+}
+
+.window-60{
+    width: 60%;
+}
+
+.window-50{
+    width: 50%;
+}
+
+
+.width-100{
+    width: 70%;
+}
+
+.width-80{
+    width: 80%;
+}
+
+.width-70{
+    width: 70%;
+}
+
+.width-60{
+    width: 60%;
+}
+
+.width-50{
+    width: 50%;
+}
+
+
+.text-filter{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding: 1rem;
+}
+
+.mycart-amount-list{
+    display: flex;
+    flex-direction:column ;
+    gap: 2px;
+}
+.mycart-amout-desc{
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+.section {
+    display: flex;
+    justify-content: center;
+    align-content: center;
+}
+
+/* my cart css  */
+
+.qty {
+    width: 40px;
+    height: 25px;
+    text-align: center;
+}
+
+.qtyplus { 
+    width:25px;
+    height:25px;
+}
+
+.qtyminus {
+   width:25px;
+   height:25px;
+}
+
+.mycart-box{
+    display: flex;
+    flex-direction:row ;
+    justify-content: center;
+    margin-top:1rem;
+    gap:20px;
+    width: 60%;
+}
+.item-bucket{
+    min-width: 30%;
+    max-width: 40%;
+}
+.mycart-total-details{
+    min-width: 30%;
+    max-width: 40%;
+}
+
+/* Login css */
+
+.login-box,
+.signup-box{
+    display: flex;
+    flex-direction:row ;
+    justify-content: center;
+    align-items: center;
```

Comment:
The `flex-direction: row` is not needed and you can use utility class for the other properties

Reply:
removed